### PR TITLE
need study more on this

### PR DESCRIPTION
**Pull request by lilixiaoxx**

### CODE REVIEW

File: starter_code/components.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    )
+  }
+}
+
+class Button extends React.Component {
+  constructor(){
+    super();
+    this.state={
+      showResult: true,
+      image:"https://1.bp.blogspot.com/-3CMTnxVnudM/VqprPp_mVEI/AAAAAAAACrk/DUuxnoCbuaw/s640/full-hd-happy-valentines-day-hd-wallpapers.jpg"
```
Comment (changamanda) [inline starter_code/components.js:16]: You don't need this, since you hard-coded the URL into Image.

File: starter_code/components.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+}
+  toggleImage(){
+    this.setState({ showResult: !this.state.showResult });
+  }
```
Comment (changamanda) [inline starter_code/components.js:21]: Nice!

File: starter_code/components.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+  render(){
+    return (
+      <div>
+        {this.state.showResult? <Image/> :null}
+        <button onClick={this.toggleImage.bind(this)}>click me</button>
+      </div>
+
```
Comment (changamanda) [inline starter_code/components.js:29]: You don't need this empty line.

File: starter_code/components.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    )
+  }
+}
+ReactDOM.render(<Button/>, document.getElementById('app-container'));
```
Comment (changamanda) [inline starter_code/components.js:33]: Nice job! Indentation is mostly good; keep paying attention to it.

**Comment by lilixiaoxx** [comment]
morning ，traffic is awful. Can u tell my team I will for a minutes
Thanks

**Comment by changamanda** [comment]
Will do!

**Pull request closed without merging**